tooling/compile-to-definitions.js

```diff
@@ -11,7 +11,7 @@ const style = {
 };
 
 // When --write is set, files will be written in place
-// Elsewise it only prints outdated files
+// Otherwise it only prints outdated files
```

Comment:
In many places we use `Elsewise`, it is British English, replaced on `Otherwise`